ale_linters/elm/make.vim

```diff
@@ -25,6 +25,7 @@ function! ale_linters#elm#make#Handle(buffer, lines) abort
                     \    'col': l:error.region.start.column,
                     \    'type': (l:error.type ==? 'error') ? 'E' : 'W',
                     \    'text': l:error.overview,
+                    \    'detail': l:error.overview . "\n\n" . l:error.details
```

Comment:
Yeah, I can see the appeal in that, it would definitely be simpler. The thing I have against that in the case of Elm, is that `elm-make` already splits the message into an overview and a detail. Any default truncation we'd implement for linters generically is never going to do as good a job across all linters.

Even for other linters that do not split their output in this way, it might be useful to make a linter-specific split in the wrapper file.